src/styles/index.scss

```
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Helvetica", sans-serif;
+  color: #060b35;
+  margin: 0;
+  --secondary-color: #616070 ;
+  --blue-color: #00acdc;
+  --white-color: #fff;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  height: 408px;
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  margin: 0;
```

```suggestion
  margin: 0;
  padding: 32px 16px 16px;
```
better to set paddings for .card instead horizontal margins for each child